### PR TITLE
Add dmca.json and check if post contains a banned author or id

### LINKED ISSUE #1662: Ban a post or user, for serve DMCA complains
Because of DMCA complains we need a way to be able to not render some content on busy.org. Ideally a list in a json where we can fit username or posts.

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1662 .

Changes:
- Check post to see if it contains a banned author or post id

### CODE REVIEW

File: src/common/constants/dmca.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_ids": [13359110, 29061561]
```
Comment (bonustrack) [inline src/common/constants/dmca.json]: it would be easier to have the posts with `author/permlink`, is authors id based too?

Comment (jm90m) [reply src/common/constants/dmca.json]: no it isnt, but yeah i can change it to author/permlink

Comment (Sekhmet) [reply src/common/constants/dmca.json]: `authors` are usernames.

**Comment by jm90m** [comment]
changed post_ids to post_urls, and changed check to check for the url that you mentioned.
```
....  
const postURL = `${post.author}/${post.permlink}`;

return _.includes(bannedAuthors, post.author) || _.includes(bannedPostURLs, postURL);
```

@bonustrack

File: src/common/constants/dmca.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "authors": [],
+  "post_urls": ["cracked-games/ltxziiz9", "bleh773/2ryjej-on-off"]
```
Comment (bonustrack) [inline src/common/constants/dmca.json]: Can you name it `posts` and put them on multiple line?

Comment (jm90m) [reply src/common/constants/dmca.json]: yup just did

**Pull request merged**